tests/cypress/cypress/pages/Article.js

```diff
@@ -0,0 +1,146 @@
+import 'cypress-iframe'
+import testData from './TestData'
+import content from './Content'
+
+class Article {
+
+    //Article page through mouse hover on admin tool bar
+    get articleLink() {
+        return cy.get("#toolbar-item-administration-tray > nav > div.toolbar-menu-administration > ul > li:nth-child(2) > ul > li:nth-child(2) > ul > li:nth-child(1) > a")
+    }
+
+    //Article Edit title bar
+    get articleEditTitle() {
+        return cy.get("#edit-title-0-value")
+    }
+
+    //Article body - edit summary
+    get articleBodyEdit() {
+        return cy.get("#edit-body-wrapper > div > div.js-form-type-textarea.js-form-item.form-item.form-type--textarea.js-form-item-body-0-value.form-item--body-0-value > div")
+    }
+
+    //Article format text
+    get articleTextFormat() {
+        return cy.get("#edit-body-0-format > div.form-item--editor-format.js-form-item.form-item.js-form-type-select.form-type--select.js-form-item-body-0-format.form-item--body-0-format > label")
+    }
+    get articleTextFormatDropdown() {
+        return cy.get("#edit-body-0-format--2")
+    }
+
+    //Article author
+    get articleAuthor() {
+        return cy.get("#edit-field-display-author-wrapper")
+    }
+
+    //Article Save as dropdown
+    get articleSaveAs() {
+        return cy.get("#edit-moderation-state-0 > div")
+    }
+    get articleSaveAsDropdown() {
+        return cy.get("#edit-moderation-state-0-state")
+    }
+
+    //Article Save button
+    get articleSave() {
+        return cy.get("#edit-submit")
+    }
+
+    //get created article title
+    get createdArticleTitle() {
+        return cy.get("body > div.dialog-off-canvas-main-canvas > div.coh-container.coh-style-focusable-content.coh-ce-6f78460f > div > article > div.coh-container.coh-style-padding-top-medium.coh-style-padding-bottom-large.coh-container-boxed > div > div > div.coh-column.coh-style-padding-bottom-small.coh-visible-ps.coh-col-ps-12.coh-visible-sm.coh-col-sm-10.coh-visible-xl.coh-col-xl-8 > h1")
```

Comment:
These element selector `body > div.dialog-off-canvas-main-canvas > div.coh-container.coh-style-focusable-content.coh-ce-6f78460f > div > article > div.coh-container.coh-style-padding-top-medium.coh-style-padding-bottom-large.coh-container-boxed > div > div > div.coh-column.coh-style-padding-bottom-small.coh-visible-ps.coh-col-ps-12.coh-visible-sm.coh-col-sm-10.coh-visible-xl.coh-col-xl-8`

In createdArticleType, createdArticleAuthor and createdArticleContent

Could we try to make one variable and pass that variable where it is required.